index.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet"  type="text/css" href="application.css">
+  </head>
+  <body>
+    <h1 class="title">Título del libro <span class="placeholder genre">(género)</span></h1>
+    <hr class="separator">
+    <div class="description">
+      <h4 class="subtitle">Autor del libro: <p class="placeholder">Nombre del autor del libro</p></h4>
+    </div>
+    <div class="description">
+      <h4 class="subtitle">Editorial: <p class="placeholder">Nombre de la editorial</p></h4>
+    </div>
+    <div class="description">
+      <h4 class="subtitle">Año de publicación: <p class="placeholder">Nombre del año de publicación</p></h4>
+    </div>
```

Comment:
I would add container for this, instead of putting directly in the body, that way you can add a litter padding to te container,  and more styles in the future.